docs/resources.md

```diff
@@ -0,0 +1,97 @@
+# Zero components
+**s3 bucket: Terraform-state:** 
+`<project>-<env>-terraform-state`
+Terraform state s3 bucket is used for storing your terraform state in a terraform [remote backend][tf-remote-backend]
+
+**dynamo db: Terraform statelock**:
+Terraform uses [state lock][tf-remote-state] to ensure actions that could contradict each others do not happen at the same time 
+
+**IAM user: `<project>-ci-user`**:
+This user is given the ECR full-access policy, our intention is you will push your container images to ECR via this IAM
+
+**Secret manager: `ci-user-aws-key`**:
+This is a credential key pair generated when creating an IAM user, we then store it in secret manager.
+
+**VPC:**
+Subnet for AWS, all our resources will be locally accessible with each others created in here
+by default everything will be under `10.10.xxx.xxx`, [see details][zero-subnet]
+
+**EKS-cluster**:
+Our main kubernetes stack, core of our infrastructure. Our business logic and application will be deployed here, Kubernetes enables ease of scaling / management of all our microservices. 
+Kubernetes will be high availability according to your [az settings][zero-az]
+
+**EKS: k8s-workers:**
+These are EC2 instances spawned according to your `ami-` image tag, any deployments and pods will be spun up in these EC2 instances. You can scale this up at a later time if your requirement increases
+
+**EKS: fluentd / amazon-cloudwatch**:
+A unified logging layer, Fluentd handles [capturing][k8s-daemonset] all log output from your cluster and routing it to various sources like Cloudwatch, Elasticsearch, etc; In our default setup logs are routed to cloudwatch-agent which saves our logs into [AWS cloudwatch][aws-cloudwatch] for easy governance. 
+- fluentd
+- cloudwatch-agent
+- aws-cloudwatch 
+
+**EKS: ingress-nginx**:
+Use Nginx as a reverse proxy and load balancer for your cluster. This will create an AWS load balancer (ELB/ALB/NLB) and whenever an ingress is created to route traffic to your application, the controller will make sure the LB is up to date and sending traffic where it needs to go. 
+
+**EKS: External-dns**:
+For any ingresses that are added to route traffic for hosts, external-dns will automatically create DNS records for those hosts and point it to the LB created by the ingress controller. This makes is extremely easy to bring up a new site at a specific domain or subdomain. [How it works][external-dns]
+
+**EKS: Cert-manager**:
+For any ingresses that specify that they need TLS, cert-manager will automatically provision a certificate using Lets Encrypt, and handle renewing it automatically on a regular basis. Alongside external-dns, this allows you to make sure your new domains are always secured using HTTPS. 
+
+To enable cert-manager to create certificate for you using [ClusterIssuer with Zero][zero-cluster-issuer], you would be able to reference it with an annotation in your ingress as follow
```

Comment:
"To enable cert-manager to create certificates for you using [ClusterIssuer with Zero][zero-cluster-issuer], you would annotate your kubernetes ingress as follows:"